skin.json

```diff
@@ -304,6 +328,13 @@
       {"name":"flexibleSpace", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":1 },
       {"name":"fullscreen", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":50 },
       {"name":"moreOptions", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":50 }
+    ],
+    "audioOnly": [
+      {"name":"volume", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":200 },
+      {"name":"skipControls", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":200 },
```

Comment:
Is this something we can address in a future PR?

Reply:
Sure, this is just a suggestion and I'm ok with doing it later or not doing it. If we do want to change this then I think we'll probably need to do it before this feature is released though, otherwise it will not be easy to change without breaking customer configs.

Reply:
Added